### PR TITLE
UPDATE #2

### PR DESCRIPTION
**Pull request by Tislom58**
- fixed buttons issue
- added button for bulk selling

### CODE REVIEW

File: space_game.py [hunk 1]
```diff
@@ -15,7 +16,7 @@
 SCREEN_HEIGHT = int(SCREEN_WIDTH * 0.8)
 
 screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
-pygame.display.set_caption("Space game v0.2.4")
+pygame.display.set_caption("Space game v0.2.3")
```
Comment (George-Doge) [inline space_game.py:19]: Space game v0.2.4

File: space_game.py [hunk 2]
```diff
@@ -485,7 +495,6 @@ def draw(self, surface):
         if event.type == pygame.KEYDOWN:
             if event.key == pygame.K_ESCAPE:
                 main_menu_instance.state = 0
-                main_menu_instance.clicked = False
```
Comment (George-Doge) [inline space_game.py:488]: you need to leave this line here

File: space_game.py [hunk 3]
```diff
@@ -171,13 +175,13 @@ def moving(self):
...
 
-            if moving_left and self.state2_rect.left >= -45:
+            if moving_left and self.state2_rect.left >= 10:
                 self.flip = True
                 self.direction = -1
                 self.state2_rect.x -= self.speed
                 energy_consumed -= 1
 
-            if moving_right and self.state2_rect.right <= SCREEN_WIDTH + 45:
+            if moving_right and self.state2_rect.right <= SCREEN_WIDTH - 10:
```
Comment (George-Doge) [inline space_game.py:184]: if this value works, it is ok

File: space_game.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 from menu import main_menu
 
 # TODO: More stations
+# TODO: MAKE PAUSE MENU BETTER
```
Comment (George-Doge) [inline space_game.py:9]: This TODO can be deleted

**Review by George-Doge — CHANGES_REQUESTED** [review]

File: space_game.py [hunk 5]
```diff
@@ -73,8 +74,11 @@
 energy_buying = False
 energy_all_buying = False
 selling = False
+selling_all = False
 shooting = False
 background_offset = 0
+first_run = True
```
Comment (George-Doge) [inline space_game.py:80]: This can be also deleted

File: space_game.py (hunk 3 above)
Comment (Tislom58) [reply space_game.py:184]: works

**Review by George-Doge — APPROVED** [review]

**Pull request merged**